src/app/material/material.module.ts

```diff
@@ -18,7 +18,10 @@ import { MatProgressBarModule } from '@angular/material/progress-bar';
 import { MatRadioModule } from '@angular/material/radio';
 import { MatSliderModule } from '@angular/material/slider';
 import { MatTooltipModule } from '@angular/material/tooltip';
-
+import { HighchartsChartModule } from 'highcharts-angular';
+import { MatBottomSheetModule } from '@angular/material/bottom-sheet';
+import { MatExpansionModule } from '@angular/material/expansion';
+import { TextFieldModule } from '@angular/cdk/text-field';
```

Comment:
Check on the ordering of imports

Reply:
Done

Reply:
no they are not, please group all the @angular